requirements-dev.txt

```diff
@@ -17,3 +17,9 @@ wheel
 
 # Requirements for updating the vendored html5lib
 hashin
+
+# Requirements for format-check most recently black==20.8b1
+black; implementation_name == "cpython" and python_version > "3.5"
+
+# Requirements for type-check most recently mypy==0.790 and mypy-extensions==0.4.3
+mypy; implementation_name == "cpython" and python_version > "3.5"
```

Comment:
Bleach doesn't support Python < 3.6, so can we nix the python version check here?

Reply:
Yep! Dropped it in an updated https://github.com/mozilla/bleach/pull/565/commits/84451282132955ffa6941b3cb0ff77a31ff0d749